.github/workflows/periodic-merge-6h.yml

```diff
@@ -15,15 +15,11 @@ on:
   workflow_dispatch:
 
 permissions:
-  contents: read
+  contents: write  # for devmasx/merge-branch to merge branches
+  pull-requests: write  # for peter-evans/create-or-update-comment to create or update comment
 
 jobs:
   periodic-merge:
-    permissions:
-      contents: write  # for devmasx/merge-branch to merge branches
-      pull-requests: write  # for peter-evans/create-or-update-comment to create or update comment
-    if: github.repository_owner == 'NixOS'
```

Comment:
I'm assuming https://github.com/NixOS/nixpkgs/pull/371216#issuecomment-2585716064 is because the `jobs.<name>.if`s on the 6h & 24h workflows were removed?